base_custom_info/static/src/js/custom_info_renderer.js

```diff
@@ -0,0 +1,157 @@
+odoo.define('base_custom_info.CustomInfoRenderer', function (require) {
+    "use strict";
+
+    var BasicRenderer = require('web.BasicRenderer');
+    var field_registry = require('web.field_registry');
+    var core = require('web.core');
+    var qweb = core.qweb;
+
+    var CustomInfoRenderer = BasicRenderer.extend({
+        init: function (parent, state, params) {
+            if (params === undefined) {
+                params = {};
+            }
+            if (params.viewType === undefined ) {
+                params.viewType = 'custom_info';
+            }
+            this._super(parent, state, params);
+            if (
+                parent !== undefined &&
+                parent.mode === 'edit' &&
+                params.mode === undefined
+            ) {
+                this.mode = 'edit';
+            }
+            this.recordWidgets = [];
+        },
+        _itemType2Object: function (type) {
+            switch (type) {
+            default:
+                return type;
+            }
```

Comment:
:thinking: :question:

Reply:
This function is no longer required, It was a first approach and I forgot to remove it